docs/UserGuide.md

```diff
@@ -384,56 +458,146 @@ Format: `c-clear`
+
+Steps to see archived employees:
+1. Type `c-archive-list` into the _Command Box_.
+2. Press "Enter" to execute.
+
+
+Outcome:
+1. The _Result Display_ will show a success message.
+2. All archived employees will be listed in _Employee Directory_ pane,.
+
+{Screenshot}
+
+#### 3.13 Unarchiving an employee : `c-unarchive`
+Unarchives the specified employee from the archived employee list.
 
 Format: `c-unarchive INDEX`
 
-* Unarchives the employee at the specified `INDEX`.
-* The index refers to the index number shown in the displayed `employees' contact details` panel.
-* The index **must be a positive integer** 1, 2, 3, ...
+* Unarchives the employee at the specified `INDEX` in the _Employee Directory_ pane.<br>
+* The `INDEX` refers to the index number shown in the current _Employee Directory_ pane.<br>
+* The `INDEX` **must be a positive integer** (e.g. 1, 2, 3, ...​).<br>
+* The `INDEX` **should not be larger than** the total number of employees listed in the current
+_Employee Directory_ pane. (e.g. There are 2 employees listed in the current _Employee Directory_ pane. The `INDEX`
+should be less or equal to 2.)<br>
 
-<div markdown="span" class="alert alert-primary">:bulb: 
+<div markdown="span" class="alert alert-primary">
+
+**:information_source: Notes about Employee Directory:**<br>
+
+* If _Employee Directory_ currently shows a list of active/unarchived employees (Archived employees are displayed with a
+ red color label, named as `[Archived]`, in front of their names. While employees in the active/unarchived employee list
+  don't have this red label.), command `c-archive-list` needs to be used first to see a list of all your archived
+  employees. Then, `c-unarchive INDEX` can be used to unarchive a specified
+   employee identified by `INDEX`.<br>
 
-**Tip:**
-Command `c-archive-list` can be used first to show a list of all archived employees' contact details. 
-Then, `c-unarchive INDEX` can be used to unarchive a specific employee.
 </div>
 
-Examples:
-* `c-archive-list` followed by `c-unarchive 2` unarchives the 2nd person in the archived employees' contact details.
+Example:
+* Let's say one of your ex-employees, Alex Yeoh, is now rehired to work in your bubble tea shop. Since he worked in
+ the store before, and you archived his data when he left last time. Now, instead of typing his information again and
+  use `c-add` command to add him into your employee directory. You can just unarchive him
+ and move his data to your employee directory by following these steps.
+
+Steps to unarchive the first employee, Alex Yeoh, in the archived employee list:
+1. Type `c-unarchive 1` into the _Command Box_.
+2. Press "Enter" to execute.
+
+
+Outcome:
+1. The _Result Display_ will show a success message with the employee's name that you have unarchived.
+2. Alex Yeoh's information will be removed from the archived employee list shown in _Employee Directory_ pane.
```

Comment:
Same issue here

Reply:
It has been changed in the latest commit. Thanks!